### PR TITLE
WIP: Multiline card element

### LINKED ISSUE #83: Add multiline card element
We have a multiline card input element on Android already. 
We will be building a similar multiline element in the iOS SDK in the coming 1-2 weeks. 

When the iOS multiline element is done, we'd like to add React Native bindings for the multiline card element.

### PR DESCRIPTION
**Pull request by arekkubaczkowski**
closes #83

### CODE REVIEW

File: example/src/screens/MultilineWebhookPaymentScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        cardStyle={inputStyles}
+        dangerouslyGetFullCardDetails
+        style={styles.cardField}
+        placeholder={{
+          cvc: '99',
+          expiration: '00',
+          number: '0231',
+          postalCode: '999',
+        }}
+        onCardComplete={(cardDetails) => {
```
Comment (thorsten-stripe) [inline example/src/screens/MultilineWebhookPaymentScreen.tsx:101]: Can we rename to `onFormComplete` to make it clear the whole form has to be complete?

File: example/src/screens/MultilineWebhookPaymentScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        keyboardType="email-address"
+        onChange={(value) => setEmail(value.nativeEvent.text)}
+        style={styles.input}
+      />
+      <CardForm
+        autofocus
+        cardStyle={inputStyles}
+        dangerouslyGetFullCardDetails
+        style={styles.cardField}
+        placeholder={{
```
Comment (thorsten-stripe) [inline example/src/screens/MultilineWebhookPaymentScreen.tsx:95]: Remove outdated

File: example/src/screens/MultilineWebhookPaymentScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        autoCapitalize="none"
+        placeholder="E-mail"
+        keyboardType="email-address"
+        onChange={(value) => setEmail(value.nativeEvent.text)}
+        style={styles.input}
+      />
+      <CardForm
+        autofocus
+        cardStyle={inputStyles}
+        dangerouslyGetFullCardDetails
```
Comment (thorsten-stripe) [inline example/src/screens/MultilineWebhookPaymentScreen.tsx:93]: remove from example

File: android/src/main/java/com/reactnativestripesdk/CardFormViewManager.kt [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+  @ReactProp(name = "postalCodeEnabled")
+  fun setPostalCodeEnabled(view: CardFormView, postalCodeEnabled: Boolean = false) {
+    view.setPostalCodeEnabled(postalCodeEnabled);
+  }
+
+  @ReactProp(name = "placeholder")
+  fun setPlaceHolders(view: CardFormView, placeholder: ReadableMap) {
+    view.setPlaceHolders(placeholder);
+  }
```
Comment (thorsten-stripe) [inline android/src/main/java/com/reactnativestripesdk/CardFormViewManager.kt:43]: These are outdated, no? Or do they work?

**Review by thorsten-stripe — CHANGES_REQUESTED** [review]
Just a couple of nits. 

Also, I think the postalCode and billing address country of the cardform are not being submitted. PTAL. Same as with the cardfield the postalcode from the cardform should take priority for being set as billing postal code on the payment method.

File: android/src/main/java/com/reactnativestripesdk/CardFormViewManager.kt (hunk 4 above)
Comment (arekkubaczkowski) [reply android/src/main/java/com/reactnativestripesdk/CardFormViewManager.kt:43]: it works on android apart from `card field` placeholder

Comment (arekkubaczkowski) [reply android/src/main/java/com/reactnativestripesdk/CardFormViewManager.kt:43]: @thorsten-stripe so please decide you we want to get rid of this until fully working

Comment (thorsten-stripe) [reply android/src/main/java/com/reactnativestripesdk/CardFormViewManager.kt:43]: Yes, as discussed please remove everything that currently doesn't work across platforms.

File: src/components/CardForm.tsx [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+      return () => {};
+    }, [inputRef]);
+
+    return (
+      <CardFormNative
+        ref={inputRef}
+        onFormComplete={onFormCompleteHandler}
+        cardStyle={{
+          backgroundColor: cardStyle?.backgroundColor,
+          disabledBackgroundColor: cardStyle?.disabledBackgroundColor,
```
Comment (thorsten-stripe) [inline src/components/CardForm.tsx:166]: Remove as it doesn't work across platforms

**Review by thorsten-stripe — CHANGES_REQUESTED** [review]
Please remove all props that don't work across platforms

**Review by thorsten-stripe — APPROVED** [review]
LGTM, thanks

**Pull request merged**